doofinder-for-woocommerce/includes/class-klaviyo-integration.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * DooFinder Klaviyo Integration methods.
```

Comment:
@davidmolinacano he seguido el ejemplo del script del add to cart pero no estoy segura de si esto funciona así, te agradecería que revisaras este PR en detalle :pray: